### PR TITLE
atftp, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by extrowerk** [comment]
Builds fine on 64 bit. Packages also.

File: net-ftp/atftp/atftp-0.7.4.recipe [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+DESCRIPTION="atftp is a client/server implementation of the TFTP protocol \
+that implements RFCs 1350, 2090, 2347, 2348, and 2349. The server is \
+multi-threaded and the client presents a friendly interface using libreadline."
+HOMEPAGE="https://sourceforge.net/projects/atftp/"
+COPYRIGHT="2000-2021 Jean-Pierre Lefebvre & Remi Lefebvre"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/atftp/files/atftp-$portVersion.tar.gz"
+CHECKSUM_SHA256="d3c9cd0d971dfc786d7a5f4055c35d4e66aafc8102ac03473ef225bdf7edb26a"
+SOURCE_FILENAME="atfp-$portVersion.tar.gz"
```
Comment (korli) [inline net-ftp/atftp/atftp-0.7.4.recipe:11]: should be atftp

Comment (Begasus) [reply net-ftp/atftp/atftp-0.7.4.recipe:11]: Will fix it later :)

Comment (Begasus) [reply net-ftp/atftp/atftp-0.7.4.recipe]: Fixed

**Pull request merged**